harvester_e2e_tests/conftest.py

```diff
@@ -247,7 +247,8 @@ def pytest_configure(config):
         ("templates", ("marker to run VM template tests")),
         ("support_bundle", ("marker to run Support Bundle tests")),
         ("settings", ("marker to run settings tests")),
-        ("upgrade", ("marker to run the upgrade tests"))
```

Comment:
can we leave this mark for future upgrade tests?

Reply:
Done